frontend/src/app/MindMap.js

```diff
@@ -8,6 +8,8 @@ import SourceView from "./SourceView";
 import NewSourceForm from "./NewSourceForm";
 import AppFooter from "./AppFooter";
 import MiniGames from "./MiniGames";
+// eslint-disable-next-line
```

Comment:
Please avoid using this type of directive. I know that you probably just needed it for some quick local development, but the final code shouldn't need any of these directives.